src/common/api/swagger.yaml

```diff
@@ -1640,7 +1643,7 @@ paths:
           name: file_name
           description: The file to be downloaded.
           type: string
-          enum: [discovery.ign, ipxe-script]
+          enum: [discovery.ign, ipxe-script, static-network-config]
```

Comment:
Are all the changes in this file relevant to this PR? If not, please revert those that aren't.

Reply:
True. I'll change to put only changes relevant for this PR